src/main/scala/com/yotpo/metorikku/code/steps/ToAvro.scala

```diff
@@ -11,20 +11,23 @@ object ToAvro {
   val message = "You need to send the following parameters to output to Avro format:" +
     "table, schema.registry.url, schema.registry.topic, value.schema.name, value.schema.namespace " +
     "Will create an entry in the schema registry under: <schema.registry.topic>-<value.schema.namespace>.<value.schema_name>"
+  private class InputMatcher[K](ks: K*) {
+    def unapplySeq[V](m: Map[K, V]): Option[Seq[V]] = if (ks.forall(m.contains)) Some(ks.map(m)) else None
```

Comment:
Why do you have to implement it?

Reply:
Will throw Metorikku exception in-case any of the InputMatcher arguments are missing...